.cspell.json

```
@@ -627,6 +629,8 @@
     "gparted",
     "gpio",
     "Gpkg",
+    "Graphviz",
+    "graphviz",
```

I remember it's case-insensitive.

```suggestion
    "graphviz",
```